src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -31,6 +33,26 @@
 /** Serves selected information from the User's Tasks Account. TODO: Implement Post (Issue #53) */
 @WebServlet("/tasks")
 public class TasksServlet extends AuthenticatedHttpServlet {
+  private final TasksClientFactory tasksClientFactory;
+
+  /** Create servlet with default TasksClient and Authentication Verifier implementations */
+  public TasksServlet() {
+    super();
```

Comment:
nit: `super()` [might not be needed](https://stackoverflow.com/questions/4090834/when-do-i-use-super).

Reply:
Removing